indico/modules/events/abstracts/__init__.py

```diff
@@ -34,6 +34,20 @@
 logger = Logger.get('events.abstracts')
 
 
+@signals.event.updated.connect
+@signals.event.contribution_created.connect
+@signals.event.contribution_updated.connect
+@signals.event.contribution_deleted.connect
+@signals.event.session_deleted.connect
+@signals.event.session_updated.connect
+@signals.event.person_updated.connect
+@signals.event.times_changed.connect
+def _clear_boa_cache(sender, obj=None, **kwargs):
+    event = (obj or sender).event_new
+    from indico.modules.events.abstracts.util import clear_boa_cache
```

Comment:
I'd have imported it as the first line of the body of the method, but no strong argument.